Tests/scripts/utils/get_modified_files_for_testing.py

```diff
@@ -158,6 +153,63 @@ def create_type_to_file(files_string: str) -> Dict[FileType, Set[str]]:
     return types_to_files
 
 
+def filter_modified_files_for_specific_marketplace_version(files_string: str, id_set: dict,
+                                                           marketplace_version: str) -> str:
+    """filter out the files in the diff list (files_string) that are not only supported in specific marketplace version
+    Args:
+        files_string (str): The modified files.
+        id_set (dict): The id set object.
+        marketplace_version (str): The marketplace version.
+    Returns:
+        string list of diff files that are supported only in marketplace_version
+    """
+    out_files_string = ''
+    for line in files_string.split("\n"):
+        if line:
+            file_status, file_path = get_status_and_file_path_from_line_in_git_diff(line)
+            # ignoring deleted files.
+            # also, ignore all files that are not in a pack
+            if file_path:
+                if (file_status in ("m", "a") or file_status.startswith("r")) and file_path.startswith("Packs/"):
+                    file_path = strip_file_path(file_path)
+                    artifact_type = get_artifact_type(file_path)
+                    for artifacts in id_set.get(artifact_type, []):
+                        for artifact_value in artifacts.values():
+                            if file_path in artifact_value.get('file_path') and \
+                                    artifact_value.get('marketplaces') == [marketplace_version]:
+                                out_files_string += f'{line}\n'
+                                break
+    return out_files_string
+
+
+def strip_file_path(file_path):
+    if file_path.endswith('.py'):
+        return file_path.rstrip('py')
+    elif file_path.endswith('_description.md'):
+        return file_path.rstrip('_description.md')
+    elif file_path.endswith('_image.png'):
+        return file_path.rstrip('_image.png')
+    elif file_path.endswith('xif'):
+        return file_path.rstrip('xif')
+    return file_path
+
+
+def get_status_and_file_path_from_line_in_git_diff(line):
+    file_status, file_path = line.split(maxsplit=1)
+    file_status = file_status.lower()
+    # Get to right file_path on renamed
+    if file_status.startswith("r"):
+        _, file_path = file_path.split(maxsplit=1)
+    return file_status, file_path
+
+
+def get_artifact_type(file_path: str):
+    artifact_type = file_path.split('/')[2]
```

Comment:
What if the change is for a file in level 1 or 2? Shouldn't we do something like:
```suggestion
    if length(file_path.split('/')) < 2:
         return None
    artifact_type = file_path.split('/')[2]
```